plugin-server/src/main/ingestion-queues/batch-processing/each-batch-webhooks.ts

```diff
@@ -139,13 +141,16 @@ export async function eachBatchHandlerHelper(
 export async function eachMessageWebhooksHandlers(
     clickHouseEvent: RawClickHouseEvent,
     actionMatcher: ActionMatcher,
-    hookCannon: HookCommander
+    hookCannon: HookCommander,
+    groupTypeManager: GroupTypeManager
 ): Promise<void> {
     if (!actionMatcher.hasWebhooks(clickHouseEvent.team_id)) {
         // exit early if no webhooks nor resthooks
         return
     }
-    const event = convertToPostIngestionEvent(clickHouseEvent)
+
+    const groupTypes = await groupTypeManager.fetchGroupTypes(clickHouseEvent.team_id)
```

Comment:
a couple of concerns:
1. Doing this means a new PG query against our write DB for all webhooks (even though it's cached), most of which don't have group analytics.
2. group analytics is a paid feature, which currently is only gated at the UI, so someone could create their groups, set this up and unsubscribe, which shouldn't work, but we'd need to grandfather supporting if we make it work now.
3. this would change restHooks data, we likely don't want to change that as there could be dependencies on it downstream

Maybe lazy load it only where needed?

Reply:
Replied in person but so we have it here too:
1. Sure - feels fine for now though
2. Agree. I don't feel super strongly either way. We can ask around internally and add an available feature check beforehand if we think it makes sense (and we can do it in a performant way)
3. Originally yes but after chatting we said maybe it actually would make sense to include it properly (but I'll make sure it isn't for now so we can follow up later)